tools/markdown-magic/README.md

```diff
@@ -91,6 +94,25 @@ Arguments:
 -   `devDependency`: Whether or not the package is intended to be installed as a dev dependency.
     -   Default: `false`.
     -   Only observed if `installation` is `true`.
+
+#### `LIBRARY_PACKAGE_README_FOOTER`
+
+Generates simple "footer" contents for a library package README.
+
+Generally recommended for inclusion at the end of the README.
+
+Notes:
+
+-   This strictly intended as a starter template to remove the need for some handwritten boilerplate.
+    You will still need to fill in semantic and usage information.
+-   This is effectively just a wrapper around lower-level templates.
```

Comment:
Might be nice to say which templates? So if someone wants to go from this to something custom but that still looks close to it, they know which other templates they'll need.